app/hero/Hero.tsx

```diff
@@ -1,23 +1,34 @@
-import { Heading, Wrap, WrapItem, Text, Box } from "@chakra-ui/react";
+import { Button, Heading, Wrap, WrapItem, Text, Box } from "@chakra-ui/react";
 import { memo } from "react";
 import { Image } from "~app/image";
 import backgroundImage from "./bg.jpg";
 
 export default memo(function Hero() {
 	return (
-		<Wrap spacing={8} justify="center" align="center" textAlign="center">
-			<WrapItem flex="1 1 0">
-				<Box>
-					<Heading as="h1" size="3xl" d="block">
-						Louise-Schroeder-Gymnasium
+		<Wrap
+			spacing={8}
+			justify="center"
+			align="center"
+			textAlign="center"
+			maxW="6xl">
+			<WrapItem flex="1 1 0" textAlign="left">
+				<Box maxWidth={400}>
+					<Heading as="h1" size="2xl" d="block">
+						Städtisches Louise-Schroeder-Gymnasium München
 					</Heading>
 					<Text fontSize="xl" mt={4}>
 						Naturwissenschaftlich-technologisches und sprachliches
-						Gymnasium in München 🍺
+						Gymnasium in München.
+					</Text>
+					<Button background="rgb(0, 119, 255)" marginTop={3}>
```

Comment:
If the default primary color doesn't fit here, perhaps we should be using a different `colorScheme` or change the theme config. Setting a string background value breaks out of chakra's whole style system and feels...wrong.